hasher-matcher-actioner/terraform/fetcher/variables.tf

```diff
@@ -61,3 +61,8 @@ variable "collab_file" {
   description = "An optional file name of ThreatExchange Collaborations objects to prepopulate. See collabs_example.json for the correct formatting"
   type = string
 }
+
+variable "config_arn" {
```

Comment:
Within a module, can we use the ARN alone to setup the hmalib.config.common module?

Can you give a pseudo-code example of how a lambda would use the ARN / additional environment variables to initialize?

Reply:
Yea, check out how the fetcher does it, I was planning on basically duplicating the same thing. I might be able to just demonstrate in a followup diff to do the fetcher.